its/src/test/java/com/sonar/it/jenkins/utility/JenkinsUtils.java

```diff
@@ -440,6 +440,10 @@ public void assertQGOnProjectPage(String jobName) {
     WebElement qg = findElement(By.className("sonar-qg"));
     assertThat(table).isNotNull();
     assertThat(qg).isNotNull();
+
+    WebElement status = findElement(qg, By.className("sonar-qg-status"));
+    assertThat(status).isNotNull();
+    assertThat(findElement(status, By.className("badge-success"))).isNotNull();
```

Comment:
Should we change the method name to highlight the fact that it is checking for success?